app/models/automation_task.rb

```diff
@@ -1,6 +1,6 @@
 class AutomationTask < MiqRequestTask
   alias_attribute :automation_request, :miq_request
-
+  has_one :container_deployment
```

Comment:
Sure thing :), we use the automation engine in order to deploy ansible playbook on selected vms, the AutomationTask object is connected to the deployment because currently thats how we know the state of the deployment and decide whether it was successful or not.
You are right that that link could be generated on the AutomationTask only in cases we use it in the context of deployment, maybe you can send me an example of such sub classing?